fvtest/gctest/GCConfigTest.cpp

```diff
@@ -37,6 +37,17 @@
 #define STRINGFY(str) DO_STRINGFY(str)
 #define DO_STRINGFY(str) #str
 
+const char *gcTests[] = {"fvtest/gctest/configuration/sample_GC_config.xml",
```

Comment:
Are you worried about adding new tests / removing tests during debugging?  If it is removing tests you can just change the filter so it is not included.  If it is adding tests then yes we have to recompile but it is better than the alternative which seems buggy and error prone.